### ISSUE TITLE
Add basic accessibility alt/aria properties onto IMG elements

### ISSUE THREAD

**Issue by ErroneousBosch** [erintheunready/ComicControl#12]
The generated img tags in this appliance lack any sort of accessibility related properties. This is a major hurdle in achieving the basic standards of accessibility. I have made a first pass at correcting this that I will attach as a patch file in a comment.

**Comment by ErroneousBosch** [comment]
Github will not let me attach patch files, but I am attaching it with a `.txt` extension.
(edit: uploaded new version which allows for more configuration)
[img_tag_accessibility.patch.3.txt](https://github.com/erintheunready/ComicControl/files/7214819/img_tag_accessibility.patch.3.txt)
